Makefile

```diff
@@ -23,6 +23,7 @@ publish:
 		echo "now must:"; \
 		echo "- seed websites: /ipfs/$$hash"; \
 		echo "- add to ipfs.io: $$ipfshash"; \
+	dnslink-deploy --domain=ipfs.io --record=@ --path=$$ipfshash
```

Comment:
maybe this line could also load the token from a file that's expected to be there, but not committed to repo.